ipv8/src/main/java/nl/tudelft/ipv8/Community.kt

```diff
@@ -64,7 +61,7 @@ abstract class Community : Overlay {
         network.blacklist.addAll(DEFAULT_ADDRESSES)
 
         job = SupervisorJob()
-        scope = CoroutineScope(Dispatchers.Main + job)
+        scope = CoroutineScope(dispatcher + job)
```

Comment:
(4/4) This will now be `Dispatchers.Default` as opposed to being `Dispatchers.Main` in the demo application.